bin/gwc/db1link.ml

```diff
@@ -1296,6 +1296,10 @@ let link next_family_fun bdir =
     begin
       if !do_consang then ignore @@ ConsangAll.compute base true ;
       Gwdb.sync base ;
+      (* FIXME ad hoc solution. wiznotes should be handled same as notes_d *)
```

Comment:
Is wiznote directory pointless if no there is no wiznote? If so, this line is a better patch for db1link waiting for proper fix as mentioned in `FIXME`

```diff
let output_wizard_notes bdir wiznotes =
  let wizdir = Filename.concat bdir "wiznotes" in
  if wiznotes <> [] then begin
+   if not @@ Sys.file_exists wizdir then Unix.mkdir wizdir 0o755 ;
    List.iter begin fun (wizid, text) ->
      let fname = Filename.concat wizdir wizid ^ ".txt" in
      write_file_contents fname text
    end wiznotes
  end
  ```